kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/admin/UserTaskAdminServiceIntegrationTest.java

```diff
@@ -971,7 +971,15 @@ private void testCancelNotify(boolean whenNotStarted) throws Exception {
             Assertions.assertThat(notifications).isNotNull();
             Assertions.assertThat(notifications).hasSize(0);
 
-            EmailNotification emailNotification = EmailNotification.builder().from("test@jbpm.org").replyTo("no-reply@jbpm.org").subject("reminder").body("my test content").users(Arrays.asList(USER_JOHN)).build();
+            EmailNotification emailNotification = EmailNotification.builder()
+                                                                   .from("test@jbpm.org")
+                                                                   .replyTo("no-reply@jbpm.org")
+                                                                   .subject("reminder")
+                                                                   .body("my test content")
+                                                                   .users(asList(USER_JOHN))
+                                                                   .emails(asList("test@jbpm.org"))
```

Comment:
Though this is a test (any value works), it would be meaningful if email here is different than the "from" (target vs origin).
If possible, use more than one email to take advantage of using "asList" instead of "Collections.singletonList".

Reply:
done